src/NuGet.Core/NuGet.Packaging/PublicAPI/netstandard2.0/PublicAPI.Unshipped.txt

```diff
@@ -1,3 +1,12 @@
 #nullable enable
+~NuGet.ContentModel.ContentPropertyDefinition.ContentPropertyDefinition(string name, System.Func<System.ReadOnlyMemory<char>, NuGet.ContentModel.PatternTable, object> parser) -> void
```

Comment:
If we're breaking the API, do we need to still keep it public?

As far as I'm concerned, I see these APIs as internal NuGet  detail.